samples/Basic_Storage/basic_storage.hw

```diff
@@ -4,6 +4,7 @@
 	"devices": {
 		// Override default (conservative) flash settings for maximum performance
 		"spiFlash": {
+			// "mode": "qio", // This doesn't always work, depends on the flash chip
```

Comment:
Replace the comment `// This doesn't always work, depends on the flash chip` with `// Make sure to set the mode according to your flash chip. Allowed SPI modes are listed here: https://sming.readthedocs.io/en/latest/_inc/Sming/Components/esptool/index.html#envvar-SPI_MODE`.